csharp/src/Google.Protobuf.Test/CodedInputStreamExtensions.cs

```diff
@@ -49,5 +49,15 @@ public static T ReadMessage<T>(this CodedInputStream stream, MessageParser<T> pa
             stream.ReadMessage(message);
             return message;
         }
+
+#if !PROTOBUF_NO_ASYNC
+        public static T ReadMessage<T>(this CodedInputStream stream, AsyncMessageParser<T> parser)
```

Comment:
Was this intended to be the async one? If not, it's not clear to me why we need this. (I can vaguely imagine, but can't quite see it - whereas I can definitely see why we'd want ReadMessageAsync.)